### ISSUE TITLE
broken docker-compose

### ISSUE THREAD

**Issue by luigisaetta** [eclipse/kapua#1869]
**Describe the bug**
Not able to start Kapua using docker-compose

**To Reproduce**
1. git clone the repo
2. go into deployments
3. run docker-deploy.sh
4. during the download of docker images  

Deploying Eclipse Kapua...
Pulling events-broker (kapua/kapua-events-broker:latest)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling events-broker (kapua/kapua-events-broker:latest)...
ERROR: pull access denied for kapua/kapua-events-broker, repository does not exist or may require 'docker login'
Deploying Eclipse Kapua... DONE!

**Comment by Coduz** [comment]
Hi @luigisaetta ,

we just realized that image `kapua/kapua-events-broker` has never been pushed to Docker Hub registry. 

I'll push all images this morning.

Sorry for the inconvenience.

Regards, 

_Alberto_

**Comment by luigisaetta** [comment]
Hi @Coduz 

don't worry, it will be fine.

Great work, go on.

**Comment by Coduz** [comment]
Hi @luigisaetta 

upon further investigation I discovered that until Kapua `1.0.0-M6` (included) we never generated that container.

Configuration of the `docker-maven-plugin` has been done in #1824 which has been merged on the 20th of June. That container will be available in Kapua `1.0.0-M7` (which will release soon). 

If you want to run the `docker-deploy.sh` script I suggest you to run a Kapua build of the `develop` branch with the following command:
`mvn clean install -Pconsole,docker` which will build containers on your machine.

Then running the `docker-deploy.sh` will pick those newly generated containers and run them.

Regards, 

_Alberto_
